bindings/matrix-sdk-ffi/Cargo.toml

```diff
@@ -14,17 +14,17 @@ publish = false
 release = true
 
 [lib]
-crate-type = ["cdylib", "staticlib"]
+crate-type = ["cdylib", "staticlib", "lib"]
```

Comment:
I'm not sure it's related to what the patch says it does. Can you split this in another patch please?